packages/angular/projects/ui-angular/theme.css

```diff
@@ -1,99 +1,2 @@
-[data-amplify-authenticator] {
-  font-size: 14px;
-  font-family: 'Amazon Ember';
-  display: inline-block;
-}
-
-[data-amplify-header] {
-  margin-top: 0px;
-}
-
-[data-amplify-fieldset] {
-  border-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-[data-amplify-container] {
-  width: 22rem;
-  padding: 3rem;
-  border-style: solid;
-  border-width: 1px;
-  border-color: rgba(0, 0, 0, 0.1);
-  background-color: white;
-}
-
-[data-amplify-form-input] {
-  border-style: solid;
-  display: block;
-  padding: 0.875rem;
-  width: 90%;
-  margin-bottom: 1rem;
-  border-color: rgba(0, 0, 0, 0.1);
-  border-width: 1px;
-  border-radius: 2px;
-}
-
-[data-amplify-form-input]:focus {
-  outline: none;
-  border: 1px solid cornflowerblue;
-}
-
-[data-amplify-label] {
-  display: block;
-  margin-bottom: 0.625rem;
-}
-
-[data-amplify-footer] {
-  margin-top: 2rem;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-}
-
-[data-amplify-button] {
-  background-color: black;
-  padding: 0.875rem 1.5rem 0.875rem 1.5rem;
-  border-style: none;
-  color: white;
-  border-radius: 2px;
-  border-width: 1px;
-  margin: 0.25rem;
-}
-
-[data-amplify-federated] [data-amplify-button] {
-  background-color: white;
-  color: black;
-  border-style: solid;
-  border-radius: 2px;
-  border-color: rgba(0, 0, 0, 0.1);
-  width: 100%;
-}
-
-[data-amplify-button]:disabled {
-  background-color: grey;
-}
-
-[data-amplify-link] {
-  color: cornflowerblue;
-  cursor: pointer;
-}
-
-[data-amplify-link]:hover {
-  text-decoration: underline;
-}
-
-[data-amplify-error] {
-  color: orangered;
-}
-
-[data-amplify-error]:last-child {
-  margin-bottom: 1rem;
-}
-
-[data-amplify-federated] {
-  display: flex;
-  flex-direction: column;
-  margin-bottom: 1rem;
-}
+@import '@aws-amplify/ui/dist/styles.css';
```

Comment:
Angular app built with `@angular/cli` has trouble finding `'@aws-amplify/ui/styles.css'`, though it is declared through `exports` and `browser` field. 

Adding `dist/` to unblock upcoming styling PRs, but I'll revisit this later (or open an issue on @angular/cli). Possibly related: https://github.com/angular/angular-cli/issues/3500